open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
<p>No first empty line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>I don't understand this comment very well, in the code ,it is:</p><p> </p><p>&lt;a name='5'&gt;&lt;/a&gt;</p><p> </p><p><strong>## 5. Creating and Training a Logistic Regression Model</strong></p><p> </p><p>We will start by importing <code>LogisticRegression</code> models available in Sklearn linear models and then proceed with model training. </p><p> </p><p>So I think there is empty line now.</p><p> </p>